### DISCUSSION TITLE
New releases?

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [madMAx43v3r/chia-plotter#607] [Q&A]
Hi I first installed this plotter on June 14th and things seem to be working out pretty well.  I'm just wondering if there are any critical bug fixes or new feature adds that have been added since then, and if I should pull the latest and recompile.  Thank you.

**Comment by madMAx43v3r** [comment]
yeah makes sense to update after one week, some new features and fixes

**Reply by jayhohoho2019** [reply]
Thank you.
